src/platform/graphics/constants.js

```diff
@@ -618,11 +618,64 @@ export const PIXELFORMAT_ATC_RGBA = 30;
  */
 export const PIXELFORMAT_BGRA8 = 31;
 
+// map of engine PIXELFORMAT_*** enums to the pixel byte size
+export const pixelFormatByteSizes = [];
```

Comment:
Kinda wish these were true Maps (you can call the Map constructor with an array for multiple entries). Not a blocker for merging this though, of course.

Reply:
arrays have slightly faster access though, so why map?

Reply:
Map works much better, thanks!

Reply:
@mvaligursky Maps are a bit more deliberate and safer type-wise, in my experience.